week.go

```diff
@@ -163,3 +163,55 @@ func FromTime(t time.Time) Week {
 	year, week := t.ISOWeek()
 	return Week{year: year, week: week}
 }
+
+// Weekday represents day of the week in ISO: Monday is 1st and Sunday is 7th
+type Weekday int
```

Comment:
I don't think that we should expose another `Weekday` type and reuse [`time.Weekday`](https://golang.org/src/time/time.go?s=10675:10691#L293) from the standard library instead.

Reply:
The reason I've added it here is that Weekday in the standard library starts on Sunday=0 and ens on Saturday=6. But ISO representation of the weekday is Monday=1 ... Sunday=7.
As this library represents ISO week format, I've added proper ISO representation.

Reply:
I see: however I still think that using types from the standard library makes it easier for users of this library and is also more idiomatic.